dev.Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM kernai/refinery-parent-images:v1.15.0-mini
+
+WORKDIR /app
+
+COPY requirements*.txt .
```

Comment:
has requirements*.txt instead of requirements-dev.txt

Reply:
That's intended, I need to copy `requirements.txt` and `requirements-dev.txt` to the image, because the dev reqs depend on the normal ones.